adoc/admin-security-ldap.adoc

```diff
@@ -555,4 +554,94 @@ base64 encoded PEM file can be generated by running:
 cat <ROOT_CA_PEM_FILE> | base64 | awk '{print}' ORS='' && echo
 ----
 
-Then, refer to <<sec-admin-security-rbac-update>> to apply the dex.yaml and <<sec-admin-security-rbac-apply>> to access through Web or CLI.
+Then, refer to <<sec-admin-security-rbac-update>> to apply the Dex `custom.yaml` and <<sec-admin-security-rbac-apply>> to access through Web or CLI.
+
+==== Example 3: Dex Active Directory TLS Connector Configuration using sAMAccountName (`addons/dex/patches/custom.yaml`)
+Configure Dex ConfigMap to use Active Directory using the following template:
+----
+connectors:
```

Comment:
Could you please ref to #918? Otherwise, the `skuba addon upgrade apply` would not work. Thanks.

Reply:
ok. seems like it was merged yesterday, so I was compared with previous example.  BTW, seems like there are other examples need to change to kustomized also.

Reply:
yes, you're right.